packages/checkout-storefront/src/fetch/requests.ts

```diff
@@ -17,82 +17,136 @@ import { PayResult } from "./types";
 import { urlJoinTrailingSlash } from "./urlJoin";
 
 export type PaymentMethodsRequestArgs = {
+  saleorApiUrl: string;
   channelId: string;
   checkoutApiUrl: string;
 };
 
 export const getPaymentMethods = ({
+  saleorApiUrl,
   checkoutApiUrl,
   channelId,
 }: PaymentMethodsRequestArgs): FetchResponse<ChannelActivePaymentProvidersByChannel> =>
-  fetch(urlJoinTrailingSlash(checkoutApiUrl, "active-payment-providers", channelId));
+  fetch(
+    urlJoinTrailingSlash(checkoutApiUrl, "active-payment-providers", channelId) +
```

Comment:
why do we have `urlJoinTrailingSlash` if we already use `url-join` package?

Reply:
`urlJoinTrailingSlash` is a wrapper around `url-join` that makes sure trailing slash is added.